### PR TITLE
WIP - Text Component by Bob

### PR DESCRIPTION
**Pull request by TheBobbyLlama**
This is Bob's version of the Text component.

### CODE REVIEW

File: src/components/Text/Text.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+const bgStyle: React.CSSProperties = {
+	backgroundColor: '#e4e4e4',
+	display: 'inline-block', // Shrink wrap
+	padding: '4px 0'
+};
+
+const rowStyle: React.CSSProperties = {
+	padding: '4px 8px'
+}
+
```
Comment (jenniferli627) [inline src/components/Text/Text.stories.tsx:18]: Maybe we can move the styles into _Text.scss file?

**Review by jenniferli627 — COMMENTED** [review]
Very good job, code works in my local server

File: src/components/Text/Text.stories.tsx (hunk 1 above)
Comment (TheBobbyLlama) [reply src/components/Text/Text.stories.tsx:18]: These styles aren't part of the component, they're only for making it look nice in Storybook.  You can see something similar done in `Button.stories.tsx`, there's just a little more styling here due to the need for a background to show the components on.

Comment (jenniferli627) [reply src/components/Text/Text.stories.tsx:18]: Got it, thanks for the clarification.

**Pull request closed without merging**